File: api/src/main/java/com/readrops/api/services/freshrss/FreshRSSCredentials.java
package com.readrops.api.services.freshrss;

import com.readrops.api.services.Credentials;

public class FreshRSSCredentials extends Credentials {

    private static final String AUTH_PREFIX = "GoogleLogin auth=";

    public FreshRSSCredentials(String token, String url) {
        super(token != null ? AUTH_PREFIX + token : null, url);

    }
}

File: api/src/main/java/com/readrops/api/services/freshrss/FreshRSSCredentials.kt
package com.readrops.api.services.freshrss

import com.readrops.api.services.Credentials

class FreshRSSCredentials(token: String?, url: String) :
        Credentials(token?.let { AUTH_PREFIX + it }, url) {

    companion object {
        private const val AUTH_PREFIX = "GoogleLogin auth="
    }
}